src/controllers/notificationController.ts

```diff
@@ -82,20 +84,23 @@ export default class NotificationController {
 
   public async createOrUpdateInstallation(
     req: express.Request
-  ): Promise<
-    | IResponseErrorInternal
-    | IResponseErrorValidation
-    | IResponseSuccessJson<SuccessResponse>
-  > {
+  ): Promise<IResponseErrorValidation | IResponseSuccessJson<SuccessResponse>> {
     return withUserFromRequest(req, async user =>
       withValidatedOrValidationError(InstallationID.decode(req.params.id), _ =>
         withValidatedOrValidationError(
           Installation.decode(req.body),
-          installation =>
-            this.notificationService.createOrUpdateInstallation(
-              user.fiscal_code,
-              installation
-            )
+          installation => {
+            // async fire & forget
+            this.notificationService
+              .createOrUpdateInstallation(user.fiscal_code, installation)
+              .catch(err => {
```

Comment:
`createOrUpdateInstallation` never catch. Check the response type to log errors.